src/Form/TypeGuesser/CronExpressionTypeGuesser.php

```diff
@@ -4,18 +4,27 @@
 
 namespace Setono\CronExpressionBundle\Form\TypeGuesser;
 
-use Brick\Reflection\ReflectionTools;
 use Cron\CronExpression;
-use ReflectionClass;
-use ReflectionException;
+use phpDocumentor\Reflection\DocBlock\Tags\Var_;
+use phpDocumentor\Reflection\DocBlockFactory;
+use phpDocumentor\Reflection\Types\ContextFactory;
+use phpDocumentor\Reflection\Types\Object_;
 use Setono\CronExpressionBundle\Form\Type\CronExpressionType;
 use Symfony\Component\Form\FormTypeGuesserInterface;
 use Symfony\Component\Form\Guess\Guess;
 use Symfony\Component\Form\Guess\TypeGuess;
 use Symfony\Component\Form\Guess\ValueGuess;
+use Symfony\Component\PropertyInfo\PropertyTypeExtractorInterface;
+use Symfony\Component\PropertyInfo\Type;
 
 final class CronExpressionTypeGuesser implements FormTypeGuesserInterface
 {
+    private ?PropertyTypeExtractorInterface $extractor;
+
+    public function __construct(?PropertyTypeExtractorInterface $extractor = null)
+    {
+        $this->extractor = $extractor;
```

Comment:
```suggestion
        $this->extractor = $extractor ?? <initialize your own property type extractor>;
```

I think this is a better approach, then you only have to deal with one dependency. The docs are here: https://symfony.com/doc/current/components/property_info.html. It looks doable

Reply:
i only added the ones used for the current tests.

there are many more type extractor, like the doctrine ones